File: Praktikum Pemrograman I Modul 4/Praktikum I Modul4  07-11-2023/Python/PRAK404-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
# perulangan
while True:
    # pilihan
    print("Pilih Program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    # input pilihan
    pil = int(input("Masukkan Pilihan : "))

    # kondisi jika input == 5 akan keluar dari perulangan 
    if (pil == 5): 
        print("Terimakasih, telah menggunakan kalkulator Akhmad Chaidar Ananda")
        break
    # kondisi jika input > 5 akan kembali melakukan perulangan dan meminta pengguna menginputkan pilihan
    elif (pil > 5): 
        print("Input anda salah, silahkan coba lagi") 
        continue
    # input nilai a dan b
    a = float(input("Masukkan nilai pertama : "))
    b = float(input("Masukkan nilai kedua   : "))

    # kondisi switch case
    match pil:
        # pilihan == 1
        case 1:
            hasil = a + b
            print(f"Hasil pertambahan antara {a:.2f} dengan {b:.2f} adalah {hasil:.2f}")
            
        # pilihan == 2
        case 2:
            hasil = a - b
            print(f"Hasil pengurangan antara {a:.2f} dengan {b:.2f} adalah {hasil:.2f}")
            
        # pilihan == 3
        case 3:
            hasil = a * b
            print(f"Hasil perkalian antara {a:.2f} dengan {b:.2f} adalah {hasil:.2f}")
            
        # pilihan == 4
        case 4:
            hasil = a / b
            print(f"Hasil pembagian antara {a:.2f} dengan {b:.2f} adalah {hasil:.2f}")

File: Praktikum Pemrograman I Modul 4/Praktikum I Modul4  07-11-2023/C/PRAK404-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>

int main(){

    // deklarasi variabel
    int pil;
    float a, b;
    float hasil;

    // perulangan 
    while (1)
    {
        // input
        printf("Pilih Program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");

        printf("Masukkan Pilihan : ");
        scanf("%d", &pil);

        // pengkondisian
        // jika pilihan == 5 akan melakukan break terhadap perulangan
        if (pil == 5){
            printf("Terimakasih, telah menggunakan kalkulator Akhmad Chaidar Ananda");
            break;
        // jika input tidak sesuai dengan pilihan, akan melakukan perulangan kembali
        } else if (pil > 5){
            printf("Input anda salah, silahkan coba lagi\n");
            continue;
        }

        // input nilai pertama dan kedua
        printf("Masukkan nilai pertama : ");
        scanf("%f", &a);
        printf("Masukkan nilai kedua   : ");
        scanf("%f", &b);

        // pengkondisian menggunakan switch case
        switch (pil)
        {
        // pilihan == 1
        case 1:
            hasil = a + b;
            printf("Hasil pertambahan  antara %.2f dengan %.2f adalah %.2f\n",a, b, hasil);
            break;
        
        // pilihan == 2
        case 2:
            hasil = a - b;
            printf("Hasil pengurangan  antara %.2f dengan %.2f adalah %.2f\n",a, b, hasil);
            break;
        
        // pilihan == 3
        case 3:
            hasil = a * b;
            printf("Hasil perkalian  antara %.2f dengan %.2f adalah %.2f\n",a, b, hasil);
            break;
        
        // pilihan == 4
        case 4:
            hasil = (float ) a / b;
            printf("Hasil pembagian  antara %.2f dengan %.2f adalah %.2f\n",a, b, hasil);
            break;

        }
        
    }
    
    return 0;
}